Gemfile

```diff
@@ -75,6 +75,7 @@ group :production do
 end
 
 group :test do
+  gem 'simplecov', '~> 0.11.0', require: false
```

Comment:
@johndbritton I'm somewhat used to fixate version numbers (at least in larged projects) to have a controlled updated process in which one can bump versions as needed step by step thus reducing the propability of introducing errors due to the version changes. But one may argue that with 99% coverage this is not really needed since one will spot introduced errors right away (with less coverage these might be go through unnoticed).

Anyhow I'll stick to your project policies so just let me know if I should remove the version and I'll will update the PR accordingly ;)